### ISSUE TITLE
Switch hardcoded spaces to \t?

### ISSUE THREAD

**Issue by JosephLandy** [dsznajder/vscode-es7-javascript-react-snippets#68]
Hey @dsznajder,

Is there any reason not to use \t in the snippets, to support different indentation levels rather than hardcoding two spaces? I saw there was a related issue a while ago that wasn't solved.
So for the test block snippet, 
```
"body": ["test('should $1', () => {", "  $0", "})", ""],
```
Would change to 
```
"body": ["test('should $1', () => {", "\t$0", "})", ""],
```
I tested it out with some snippets and it seems to work fine. \t just gets substituted to the number of spaces defined in the VS Code settings.

 I could probably change it in the next little while, if you would like :)

**Comment by dsznajder** [comment]
@JosephLandy If it solves people problem, of course, I would like :) I've just done normal spaces as it was easier for me to define those

**Comment by dsznajder** [comment]
@JosephLandy Released in `2.3.0`

**Comment by JosephLandy** [comment]
Thanks!
